src/main.ts

```diff
@@ -35,8 +35,9 @@ async function bootstrap() {
   );
   app.setGlobalPrefix("api");
 
+  app.enableCors();
   await app.listen(3000);
-
+  console.log(`Backend running: http://localhost:3000/`);
```

Comment:
and we can definitely add some console logs here to make it clear that the backend is running on `localhost:5173/api` AND `localhost:3000`